src/test/java/blackjack/domain/player/DealerTest.java

```diff
@@ -0,0 +1,103 @@
+package blackjack.domain.player;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardNumber;
+import blackjack.domain.card.CardPattern;
+import blackjack.domain.card.Cards;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class DealerTest {
+
+    @Test
+    @DisplayName("딜러를 정상 생성 한다.")
+    void createDealer() {
+        assertThatCode(() -> new Dealer(new Cards(firstDrawTwoCards())))
+                .doesNotThrowAnyException();
+    }
+
+    @Test
+    @DisplayName("딜러는 처음 받은 카드 중 하나만 보여준다.")
+    void openFirstCards() {
+        final List<Card> firstDrawTwoCards = firstDrawTwoCards();
+        Dealer dealer = new Dealer(new Cards(firstDrawTwoCards));
+        final Card expected = firstDrawTwoCards.get(0);
+
+        final Card actual = dealer.openFirstCards().get(0);
+
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("딜러는 카드를 받아서 본인의 카드 리스트에 추가할 수 있다.")
+    void receiveCard() {
+        Cards cards = new Cards(firstDrawTwoCards());
+        Dealer dealer = new Dealer(cards);
+        final int expected = 3;
+
+        dealer.receiveCard(new Card(CardPattern.DIAMOND, CardNumber.SIX));
+        final int actual = dealer.cards.getCards().size();
+
+        assertThat(actual).isEqualTo(expected);
+    }
```

Comment:
```suggestion
    @Test
    @DisplayName("딜러는 카드를 받아서 본인의 카드 리스트에 추가할 수 있다.")
    void receiveCard() {
        Cards cards = new Cards(firstDrawTwoCards());
        Dealer dealer = new Dealer(cards);
        Card card = new Card(CardPattern.DIAMOND, CardNumber.SIX);

        dealer.receiveCard(card);

        assertThat(dealer.getCards().getCards()).contains(card);
    }
```

`contains`를 이용해서 비교하면 어떨까요?

Reply:
새로운 방법 감사합니다! 어떤 테스트인지 훨씬 알기 쉬운 것 같아요😃